mmedit/models/editors/inst_colorization/color_utils.py

```diff
@@ -0,0 +1,307 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
Maybe we can move colorspace transformation functions in this file to MMCV later. (Maybe can convert to a **good first issue**)

Reply:
Agree.